layouts/about-release-schedule.hbs

```diff
@@ -14,7 +14,7 @@
         {{{ contents }}}
 
         <p>
-          <img src="https://raw.githubusercontent.com/nodejs/Release/master/schedule.svg?sanitize=true" alt="{{ title }}" width="760" height="396">
+          <img src="https://github.com/nodejs/Release/blob/main/schedule.svg" alt="{{ title }}" width="760" height="396">
```

Comment:
I don't think you can just reference the URL if you want it to display an image. I think it needs to use the githubusercontent URL. I the comment was about changing the branch part

Reply:
Oops……fixed, sorry.